docker/ubuntu18/openvino_tensorflow_cgvh_runtime_2.1.0-serving.dockerfile

```diff
@@ -16,9 +16,9 @@ ENV DEBIAN_FRONTEND=noninteractive
 
 SHELL ["/bin/bash", "-xo", "pipefail", "-c"]
 
-ARG TF_VERSION="v2.8.0"
```

Comment:
If this dockerfile is not working with TF-2.9.1 and OVTF-2.1.0, then suggest to restore it to its previous version.